go/test/mockllm/server.go

```diff
@@ -13,6 +13,7 @@ import (
 
 // Server is the main mock LLM server
 type Server struct {
+	Address           string
```

Comment:
Nit, I think this should be a config option rather than settable on the `Server` struct. Otherwise the user has to know to modify the structure after creation.